go-controller/pkg/ovn/service.go

```diff
@@ -165,6 +167,130 @@ func (ovn *Controller) syncServices(services []interface{}) {
 	}
 }
 
+func (ovn *Controller) createService(service *kapi.Service) error {
+	klog.V(5).Infof("Creating service %s", service.Name)
+	if !util.IsClusterIPSet(service) {
+		return fmt.Errorf("no cluster IP for service %s found", service.Name)
```

Comment:
Why is this an error message? It should be `\klog.V(5).Infof("Skipping service %s due to clusterIP = %q",
			svc.Name, svc.Spec.ClusterIP)` and we should just return `nil`, right?

Reply:
Yeah for sure we need to ignore services without a ClusterIP. This is perfectly valid and we already ignored them, it shouldn't be an error.

Reply:
Agree. Should be valid for headless service. I'll fix it. Thanks.